enterprise/server/sociartifactstore/sociartifactstore.go

```diff
@@ -327,12 +328,17 @@ func (s *SociArtifactStore) getArtifactsFromCache(ctx context.Context, imageConf
 }
 
 func (s *SociArtifactStore) pullAndIndexImage(ctx context.Context, imageRef ctrname.Digest, configHash v1.Hash, credentials *rgpb.Credentials) (*repb.Digest, []*repb.Digest, error) {
-	log.Infof("soci artifacts not found, generating them for image %s", imageRef.DigestStr())
+	groupId := getGroupIdForDebugging(ctx, s.env)
+	log.Infof("soci artifacts not found, generating them for group %s / image %s", groupId, imageRef.DigestStr())
```

Comment:
this is fine, just for future reference another way to handle this is to call

ctx = log.EnrichContext(ctx, "group_id", getGroupId())

somewhere high up in the chain of processing here when you have groupID, and then these log statements can become:

log.CtxInfo("soci artifacts not found, generating for image: %s", imageStr)

and the log will include the groupID you attached above.